content/docs/get-started/index.md

```diff
@@ -59,43 +63,39 @@ save(
 )
 ```
 
-Here, we loaded a well-known
-[Iris flower dataset](https://archive.ics.uci.edu/ml/datasets/iris) with
-`scikit-learn` and trained a simple classifier. Instead of pickling the model,
-we persisted it using MLEM's `save` API.
+Here, we loaded a well-known [Iris flower dataset] with scikit-learn and trained
+a simple classifier (see the full list of [supported ML frameworks]). We then
+captures it using `mlem.api.save`, which creates the following files:
 
-`scikit-learn` is just an example of many supported ML frameworks. Check out the
-[full list here](/doc/object-reference/model).
-
-Now, let's see what MLEM saved by using the `tree` terminal command:
-
-```cli
-$ tree models/
+```
 models
 ├── rf
 └── rf.mlem
 ```
 
-Alongside the model binary `models/rf`, MLEM saved an additional metadata file
-`models/rf.mlem`. We refer to this as a "Codification" of the model. This
-`.mlem` "metafile" contains all the information we need in order to use the
-model later:
+Alongside the model binary `models/rf`, MLEM saved metafile `models/rf.mlem`,
+which contains all the information needed to use the model later:
 
 1. Model methods: Like `predict` and `predict_proba`
 2. Input data schema: Describes the dataframe (Iris dataset)
 3. Python Requirements: `sklearn` and `pandas` in this case, with the specific
    versions used to train the model
 
-<admon type='tip'>
+<admon type="info">
```

Comment:
💡 was looking more appealing to me than ℹ️ . Just complaining, if that's our standard for all docs websites, please ignore

Reply:
Well, is this a tip or is it useful information? The way each one looks could be changed, but that's a design question IMO. No strong opinion though, feel free to revert this if you prefer.